### ISSUE TITLE
 Keyboard focus outline shows up for disabled ranges of candlestick chart

### ISSUE THREAD

**Issue by franklinefrancis** [highcharts/highcharts#15415]
#### Expected behaviour
Keyboard focus outline does not show up for disabled ranges of candlestick chart

#### Actual behaviour
Keyboard focus outline does not show up for disabled ranges of candlestick chart

![Candlestick Range](https://user-images.githubusercontent.com/1227574/112712545-57bc7680-8ef6-11eb-9f79-b862dc135288.png)

#### Live demo with steps to reproduce
https://jsfiddle.net/franklinefrancis/k8d594j0/

#### Product version
8.2.0

#### Affected browser(s)
Chrome, Firefox

**Comment by KacperMadej** [comment]
Hi @franklinefrancis 

Thank you for reporting the issue.

The buttons could be disabled or removed. If you don't want to be able to tab through them then change [`buttons`](https://api.highcharts.com/highstock/rangeSelector.buttons) option and remove unused buttons.

The buttons that are disabled should be focusable for full a11y support (here is why - https://ux.stackexchange.com/questions/103239/should-disabled-elements-be-focusable-for-accessibility-purposes/103246#103246).

Please give us your point of view on this @franklinefrancis

**Comment by franklinefrancis** [comment]
Agreed @KacperMadej.
